docs/product-portal/accounts/metrics.md

```diff
@@ -45,6 +45,7 @@ By following these instructions you provide both of our teams with data needed t
 |`utm_content`|Used to track the name of an A-B test|`my-experiment`|<!--begin-validation-utm_content-->^[\w\/.%-]+$<!--end-validation-utm_content-->|TBD|
 |`utm_term`|Used to track the cohort or variation in an A-B test|`my-experiment-var-a`|<!--begin-validation-utm_term-->^[\w\/.%-]+$<!--end-validation-utm_term-->|TBD|
 |`utm_medium`|What type of link was used to direct to the page, if it came through a marketing campaign|`email`, `cpc`|<!--begin-validation-utm_medium-->^[\w\/.%-]+$<!--end-validation-utm_medium-->|TBD|
+|`context`|Not relevant to metrics, but this is **required** to be set to `fx_desktop_v3` if `service=sync`.|`fx_desktop_v3`|<!--begin-validation-context-->^[0-9a-z_-]+$<!--end-validation-context-->/|NA|
```

Comment:
It's only required for Fx Desktop, Fennec can set `fx_fennec_v1` and for iOS it can be `fx_ios_v1`.

Reply:
so does this mean that if a user on (e.g.) Android opens an email with a registration link that sets the parameter to the desktop value, won't get signed in properly on fennec?

Reply:
> Android opens an email with a registration link that sets the parameter to the desktop value, won't get signed in properly on fennec?

A user of Fennec *should* be alright (I haven't tested this), though they will get the experience that Desktop gets, some things might not make total sense. 

OTOH, a user of Firefox for iOS will not be. 

Fennec and Firefox Desktop use the same underlying communication mechanism to communicate between the browser and FxA (WebChannels), whereas Firefox for iOS uses a different, incompatible, scheme (CustomEvents). 

A user who opens a link with `context=fx_desktop_v3` in Firefox for iOS, or even Safari or Chrome, will have a bad time.